Commands/ConfigTrait.php

```diff
@@ -46,12 +46,22 @@ protected function renderConfig(OutputInterface $output, array $config)
 
     /**
      * @param InputInterface $input
-     * @return null|string
+     * @param bool $create
+     * @return string
+     * @throws \Exception
      */
-    protected function getConfigPath(InputInterface $input)
+    protected function getConfigPath(InputInterface $input, $create = false)
     {
+        $configOption = $input->getOption('config');
+        if ($configOption && !file_exists($configOption)) {
+            if ($create) {
+                file_put_contents($configOption, json_encode([]));
```

Comment:
If You want to create new config file with default values You may use it right now as:
`./bin/ppm config -c new_config.json`
If file doesn't exist it will create one with default values. IMO it's valid use case. If file exists it will work as usual - You can change config with command arguments.
It will only work with `Config Command`. Other commands will fail with Exception.